ssl/t1_lib.c

```diff
@@ -2669,7 +2670,21 @@ int tls12_check_peer_sigalg(SSL_CONNECTION *s, uint16_t sig, EVP_PKEY *pkey)
         if (pkeyid == EVP_PKEY_RSA)
             pkeyid = EVP_PKEY_RSA_PSS;
     }
+
+    /* Is this code point available and compatible with the protocol */
     lu = tls1_lookup_sigalg(SSL_CONNECTION_GET_CTX(s), sig);
+    if (lu == NULL)
```

Comment:
The checks for `lu != NULL` and `lu == NULL` done further down are now redundant with this change